pkg/controller/plan/scheduler/vsphere/scheduler.go

```diff
@@ -0,0 +1,117 @@
+package vsphere
+
+import (
+	"context"
+	liberr "github.com/konveyor/controller/pkg/error"
+	api "github.com/konveyor/forklift-controller/pkg/apis/forklift/v1alpha1"
+	"github.com/konveyor/forklift-controller/pkg/apis/forklift/v1alpha1/plan"
+	plancontext "github.com/konveyor/forklift-controller/pkg/controller/plan/context"
+	"github.com/konveyor/forklift-controller/pkg/controller/provider/web/vsphere"
```

Comment:
One other consideration.  The pattern I have been following is to import the web (or model) resources within provider specific packages as just: _web_ or _model_ for clarity since it's already determined to be provider (type) specific.